src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -22,28 +23,29 @@
  */
 public class LogicManager implements Logic {
     public static final String FILE_OPS_ERROR_MESSAGE = "Could not save data to file: ";
-    private final Logger logger = LogsCenter.getLogger(LogicManager.class);
 
-    private final Model model;
-    private final Storage storage;
-    private final AddressBookParser addressBookParser;
+    private static final Logger logger = LogsCenter.getLogger(LogicManager.class);
```

Comment:
why add `static`? 😯

Reply:
The logger variable itself doesn't have to be static in the current code. Was using the logger in the static method to debug previously.